secretmanager/snippets/regional_samples/snippets_test.py

```diff
@@ -271,6 +277,117 @@ def test_get_regional_secret_version(
     assert secret_id in version.name
     assert version_id in version.name
 
+def test_iam_grant_access_with_regional_secret(
+    regional_client: secretmanager_v1.SecretManagerServiceClient,
+    regional_secret: Tuple[str, str, str, str],
+    iam_user: str,
+) -> None:
+    project_id, location_id, secret_id, _ = regional_secret
+    policy = iam_grant_access_with_regional_secret.iam_grant_access_with_regional_secret(
+        project_id, location_id, secret_id, iam_user
+    )
+    assert any(iam_user in b.members for b in policy.bindings)
+
+
+def test_iam_revoke_access_with_regional_secret(
+    regional_client: secretmanager_v1.SecretManagerServiceClient,
+    regional_secret: Tuple[str, str, str, str],
+    iam_user: str,
+) -> None:
+    project_id, location_id, secret_id, _ = regional_secret
+    policy = iam_revoke_access_with_regional_secret.iam_revoke_access_with_regional_secret(
+        project_id, location_id, secret_id, iam_user
+    )
+    assert not any(iam_user in b.members for b in policy.bindings)
```

Comment:
nit: `assert all(iam_user not in b.members for b in policy.bindings)` is a bit more Pythonic.